core/src/components/button/button.ios.scss

```diff
@@ -212,3 +212,12 @@
 :host(.button-solid.ion-color.ion-activated) .button-native::after {
   background: #{current-color(shade)};
 }
+
+
+// Activated Button in Toolbar
+// --------------------------------------------------
+
+:host(.button-outline.ion-activated.in-toolbar:not(.ion-color):not(.in-toolbar-color)) .button-native {
```

Comment:
Let's add a comment that this needs to be within this file instead of `button.scss` else it gets overridden. That way we don't end up moving it back.